runtime/compiler/env/VMJ9.cpp

```diff
@@ -8254,7 +8254,7 @@ TR_J9SharedCacheVM::canMethodExitEventBeHooked()
 bool
 TR_J9SharedCacheVM::methodsCanBeInlinedEvenIfEventHooksEnabled()
    {
-   return true;
+   return !TR::Options::getCmdLineOptions()->getOption(TR_FullSpeedDebug);
```

Comment:
We need to change the definition at JITServer which now reads:
```
bool
TR_J9SharedCacheServerVM::methodsCanBeInlinedEvenIfEventHooksEnabled()
   {
   return true;
   }
```
The complication is that we are returning the value from the global options, so I am proposing to add a `comp` parameter which can be used to retrieve the options: `comp->getOption(TR_FullSpeedDebug)`

Reply:
Addressed in https://github.com/eclipse/openj9/pull/11554/commits/6e15af8fa2a907dea4f78cda8bf953e2c0af41a5